gradle.properties

```diff
@@ -1,6 +1,5 @@
 version=8.5.0
-iexecCommonVersion=8.5.0
-iexecCommonsPocoVersion=4.1.0
-
```

Comment:
I think you should keep the blank line, it appears to be a convention in all other files

Reply:
IntelliJ auto reformating ....

Reply:
Resolved in [2f50f32](https://github.com/iExecBlockchainComputing/iexec-result-proxy/pull/146/commits/2f50f32e70bcadf3fdf2dec60ce913012b2cf173)